api4/job.go

```diff
@@ -35,6 +37,37 @@ func getJob(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	forceDownload, _ := strconv.ParseBool(r.URL.Query().Get("download"))
+	if forceDownload {
+		fileName := "actiance_export.xml"
+		fileMime := "application/xml"
+		if job.Data["export_type"] == "csv" {
+			fileName = "csv_export.zip"
+			fileMime = "applicaiton/zip"
+		}
+
+		filePath := "export/" + job.Id + "/" + fileName
+		data, readFileErr := c.App.ReadFile(filePath)
+		if readFileErr != nil {
+			c.Err = readFileErr
+			return
+		}
+
+		fileReader, err := c.App.FileReader(filePath)
+		if err != nil {
+			c.Err = err
+			c.Err.StatusCode = http.StatusNotFound
```

Comment:
I'm not sure it would ever get here. As wouldn't line 50 fail first.

Reply:
Done!